docs/src/main/tut/json.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title: JSON handling
+---
+
+## Pick a library
+
+### Argonaut
+
+```scala
+libraryDependencies += Seq(
+  "org.http4s" %% "http4s-argonaut" % "0.15.0a-SNAPSHOT",
+  "com.github.alexarchambault" %% "argonaut-shapeless_6.1" % "1.1.1" // or 1.1.0 for non-a versions
+)
+```
+
+### Circe
+
+```scala
+libraryDependencies += "org.http4s" %% "http4s-circe" % "0.15.0a-SNAPSHOT"
+```
+
+### Json4s
+
+```scala
+libraryDependencies += "org.http4s" %% "http4s-json4s" % "0.15.0a-SNAPSHOT"
```

Comment:
This is an abstract module, and needs json4s-jackson or json4s-native to be useful.  json4s-native's future is in doubt (#685).